scripts/install_environment.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 set -euo pipefail
+source scripts/utils.sh
```

Comment:
i think i miss what function from utils you are using here?

Reply:
Good catch, probably it's not needed. @ori-amizur it it OK to remove?

Reply:
Yes